lib/gamera/modules/visitable.rb

```diff
@@ -20,7 +20,7 @@ def visit(fail_on_redirect = true)
     # @param wait_time_seconds [Integer] How long to wait for the correct page to load
     # @return [Boolean] true if the url loaded in the browser matches the url_matcher pattern
     # @raise [NoUrlMatcherForPage] if there's no url_matcher for this page
-    def displayed?(wait_time_seconds = Capybara.default_wait_time)
+    def displayed?(wait_time_seconds = Capybara.default_max_wait_time)
```

Comment:
`default_max_wait_time` does not exist in Capybara 2.4.4:
https://github.com/jnicklas/capybara/blob/2.4.4/lib/capybara.rb

So this PR at least needs to include a corresponding change to `gamera.gemspec`.
@jillianrosile, @asimy, am I right that we're not ready to upgrade our other projects to 2.5 yet?  This might be an opportunity to practice our own deprecation warnings.  But we're also young enough as a project to play this less formally.